apps/dashboard/app/page.tsx

```diff
@@ -1,18 +1,53 @@
 import { getServerSession } from "next-auth"
 
+import { Box } from "@mui/joy"
+
 import { authOptions } from "./api/auth/[...nextauth]/route"
 
 import PriceContainer from "@/components/price-container/price-container"
 import ContentContainer from "@/components/content-container"
+import { fetchAllTransactionsByCount } from "@/lib/get-all-transactions"
+import TransactionChart from "@/components/chart"
 
 export default async function Home() {
   const session = await getServerSession(authOptions)
+
+  const token = session?.accessToken
+  if (!token || typeof token !== "string") {
+    throw new Error("invalid token")
+  }
+
+  const btcWallet = session.userData.data.me?.defaultAccount.wallets.find(
+    (wallet) => wallet.walletCurrency === "BTC",
+  )
+  const usdWallet = session.userData.data.me?.defaultAccount.wallets.find(
+    (wallet) => wallet.walletCurrency === "USD",
+  )
+
+  const response = await fetchAllTransactionsByCount({
+    token,
+    fetchCount: 5, // Maximum fetch rotation count if set to 5; the API will be called a maximum of 5 times, which gives 500 transactions.
+  })
+
   const walletDetails = session?.userData?.data?.me?.defaultAccount?.wallets || []
 
   return (
     <main>
       <ContentContainer>
-        <PriceContainer walletDetails={walletDetails}></PriceContainer>
+        <Box
+          sx={{
+            maxWidth: "90em",
+            width: "100%",
+            margin: "0 auto",
+          }}
+        >
+          <PriceContainer walletDetails={walletDetails}></PriceContainer>
```

Comment:
`PriceContainer` is a misleading name. Pricing would typically imply something like bitcoin historical price, or current bitcoin market price, there is none of that here. 

It should be something like `WalletBalanceContainer`. similarly for this "naming debt", maybe let's do a PR and get this balance history rebased from it.